### PR TITLE
Issue 34 revision

### PR DESCRIPTION
**Pull request by escottalexander**
style.css file added with proper font sizing.
Also reference to style.css added to index.html

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
+/*
```
Comment (andy5995) [inline assets/css/style.css]: This file should be in /jekyll/assets/css/style.css It gets copied to the correct folder when the site is built.

File: index.html [hunk 2]
```diff
@@ -13,29 +13,32 @@
...
     <!-- Latest compiled and minified CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
     <!-- Font Awesome references   -->
     <link rel="stylesheet" href="http://localhost:4000/assets/css/font-awesome.min.css">
-    
-      
+
+    <!-- Custom CSS file references   -->
+    <link rel="stylesheet" href="http://localhost:4000/assets/css/style.css">
```
Comment (andy5995) [inline index.html]: This URL should be "{{ site.url }}/assets/css/style.css"

I see you copied from the line above it. That must be an error in our set-up. Me or @erichoog will have to figure out the case of that.

Comment (andy5995) [reply index.html]: actually this file shouldn't even be edited. It'll get overwritten. That change should go into 
https://github.com/mhwkb/mhwkb.github.io/blob/master/jekyll/_includes/head.html

**Comment by erichoog** [comment]
Replaced by #41

**Pull request closed without merging**